assets/js/common/TimeSeriesLineChart/TimeSeriesLineChart.jsx

```diff
@@ -0,0 +1,209 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Chart as ChartJS,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Tooltip,
+  TimeScale,
+  LogarithmicScale,
+  Legend,
+} from 'chart.js';
+import { Line } from 'react-chartjs-2';
```

Comment:
We could move the component import from the external in the end of the imports like in other component files ? 
What do you think?

Reply:
done!